### PR TITLE
Gitpractice#2 approve approve approve

### PR DESCRIPTION
**Pull request by dasbeerboot**
Hurry and approve

### CODE REVIEW

File: gitpractice02
Comment (youngjoon-lee) [inline gitpractice02]: please add an exclamation mark at the end of this line. :-1:

**Review by youngjoon-lee — CHANGES_REQUESTED** [review]
I left a comment. Please apply it before merging this PR.
Because I requested changes, we cannot merge this PR before I approve it. This is a nice feature of Github.

File: gitpractice02
Comment (dasbeerboot) [reply gitpractice02]: why

Comment (dasbeerboot) [reply gitpractice02]: wtf

Comment (youngjoon-lee) [reply gitpractice02]: just for a practice

Comment (youngjoon-lee) [reply gitpractice02]: Do it if you don't wanna be fired.

Comment (youngjoon-lee) [reply gitpractice02]: I just checked your new commit. Fine by me.

**Review by youngjoon-lee — APPROVED** [review]
LGTM :+1: 
(abbreviation of Looks Good To Me)

File: gitpractice02
Comment (dasbeerboot) [reply gitpractice02]: Then remove your red sign man

**Comment by dasbeerboot** [comment]
> LGTM 👍
> (abbreviation of Look Good To Me)

Ta

**Comment by makeajourney** [comment]
I don't know what you want me to review! 🤔

**Review by makeajourney — COMMENTED** [review]
I don't know what you want me to review! 🤔
Anyway, LGTM.
You can merge it.

**Review by makeajourney — APPROVED** [review]

**Pull request merged**
